compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/native/ComposeLayer.jsNative.kt

```diff
@@ -129,6 +132,10 @@ internal class ComposeLayer(
         scene.constraints = Constraints(maxWidth = width, maxHeight = height)
     }
 
+    internal fun getActiveFocusRect():Rect? {
```

Comment:
nit: the class itself is internal, so we can omit internal here

Reply:
done